### PR TITLE
ECDC-2588: chopper mapping bug fix

### PR DESCRIPTION
**Pull request by kmurica**

### CODE REVIEW

**Comment by kmurica** [comment]
@mattclarke could you have a look if this implementation makes sense before I create a patchset in Gerrit?
Not sure if this is the correct way to approach the problem.

**Comment by mattclarke** [comment]
Can you remind me what the context of the problem is?

File: nicos_ess/devices/epics/extensions.py [hunk 1]
```diff
@@ -66,6 +66,15 @@ def _readRaw(self, maxage=0):
     def _startRaw(self, target):
         EpicsDigitalMoveableEss.doStart(self, target)
 
+    def doStart(self, value):
+        if value in self.mapping.keys():
```
Comment (mattclarke) [inline nicos_ess/devices/epics/extensions.py:70]:
```suggestion
        if value in self.mapping:
```
Does that work? I think the mapping is either a dict or inherits from dict.

**Comment by kmurica** [comment]
moved to gerrit

**Pull request closed without merging**